src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -1,4 +1,6 @@
 import * as React from "react";
+import {Wrapper} from "./Wrapper";
+import {BuildSnapshot} from "./BuildSnapshot";
```

Comment:
Would it make sense to order these alphabetically?

Reply:
Perhaps - I'll go ahead and reorder them before merging.